interface-elements-for-web/articles/spreadsheet/data-presentation/filtering.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Filtering
+---
+# Filtering
+You can use filtering commands on the **Data** tab in the **Sort & Filter** group to activate filtering, reapply a filter, or remove all the filters in a worksheet.
+
+![EUD_ASPxSpreadsheet_DataMenu](../../../images/img26346.png)
+
+To enable filtering, focus the cell you can to filter and click **Filter**.
+
+Once filtering is turned on, click an arrow in the column header to choose a filter for the column.
+
+![EUD_ASPxSpreadsheet_FilteredTable](../../../images/spreadsheet-data-presentation-filtering-table.png)
+
+## Apply a Filter
+To filter column by particular values, click an arrow in the column header and select **Filter by Values...**. A dialog box with a list of column values appears. 
```

Comment:
an arrow? any?

Reply:
A dialog box with a list of column values appears. -> This invokes a dialog box with a list of column values.